webplugin/template/chat.html

```diff
@@ -48,8 +49,9 @@
                 if (skipRouting){
                     defaultSettings["skipRouting"] = !!skipRouting;
                 }
-                
+
                 var kommunicateSettings = {
+                    "isInputFocused" : isInputFocused,
```

Comment:
if focused is true, that means "do not focus", or that means focus it?

Reply:
Made some changes, we will instruct the customers to add param noFouced = true, which means do not focus